src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,66 @@
+/**
+ * D header file to interface with the
+ * $(HTTP pubs.opengroup.org/onlinepubs/9699919799/basedefs/aio.h.html, Linux aio API).
+ * Available since Linux 2.6
```

Comment:
Shouldn't you have dropped the Linux references here?

Take a look any of the other `core.sys.posix` module headers, eg: https://github.com/dlang/druntime/blob/master/src/core/sys/posix/pwd.d